docs/security/authorization/spark/install.md

```diff
@@ -74,6 +74,25 @@ for pointing to the right Ranger admin server.
 
 </configuration>
 ```
+- Additional configs for using Marcos in row-level filter
+  - Macros are now supported for using user/group/tag in row-filter condition expression, introduced in [Ranger 2.3](https://cwiki.apache.org/confluence/display/RANGER/Apache+Ranger+2.3.0+-+Release+Notes). This feature helps significantly simplify the row-filter condition expression by replacing explicit condition query by using user/group/tag's attributes. 
+  - Consider user `bowenliang123` with an attribute `born_city` of value `Guangzhou `, the row filter condition as `city='${{USER.born_city}}'` will be transformed to `city='Guangzhou'` in execution plan.
+  - More supported marcros and usage refer to [RANGER-3605](https://issues.apache.org/jira/browse/RANGER-3605) and [RANGER-3550](https://issues.apache.org/jira/browse/RANGER-3550).
+  - Add the follow configs to `ranger-spark-security.xml` to enable UserStore Enricher which is required by macros and scripts.
```

Comment:
it‘s not a good idea to use list here

Reply:
Which part puzzled you. or let's skip the example or the mention of ranger issue?

Reply:
any better styling suggestion here?

Reply:
plain text enough

Reply:
OK, let‘s skip the line breaks in line 78 to 80. But I prefer to keep line 81 as an item of the list. what do you think?